css/style.css

```diff
@@ -0,0 +1,649 @@
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* general */
+
+body {
+  font-size: 14px;
+  line-height: 1.5;
+  font-family: Helvetica Neue, Arial, sans-serif;
+  color: #555;
+  margin: 0;
+  padding: 0;
+}
+
+main {
+  margin-bottom: 0;
+}
+
+.lists a,
+.inside,
+.inside-3rd-section {
+  color: #fff;
+}
+
+footer {
+  text-align: center;
+  background-color: #131313;
+  padding-bottom: 20px;
+}
+
+.footer-icons {
+  background-color: #000;
+}
+
+.footer-icons ul {
+  list-style: none;
+}
+
+.link-icons a {
+  color: #fff;
+  margin-left: -10px;
+}
+
+.reach-out a,
+.footer-bottom {
+  color: #aaa;
+  margin-left: 7px;
+}
+
+.latest-list li a {
+  text-decoration: none;
+  color: #555;
+  font-size: 14px;
+}
+
+.footer-icons ul li a {
+  text-decoration: none;
+}
+
+.footer-icons ul .link-icons a {
+  margin-right: 17px;
+}
+
+.reach-out {
+  margin-top: 25px;
+  margin-bottom: 25px;
+  padding-bottom: 20px;
+  font-size: 11px;
+}
+
+.link-icons {
+  padding-top: 30px;
+  font-size: 16px;
+  margin-left: 20px;
+}
+
+.icon-cicle {
+  background-color: rgb(82, 77, 77);
+  padding: 8px;
+  border-radius: 60%;
+}
+
+.footer-bottom {
+  width: 80%;
+  margin-left: 70px;
+}
+
+.inside {
+  position: relative;
+  top: -160px;
+  left: 15px;
+  width: 90%;
+}
+
+.inside-heading {
+  font-size: 1.5em;
+  line-height: 1;
+  font-weight: 800;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.first-section {
+  margin-top: 0;
+  margin-bottom: -130px;
+}
+
+.section-h2,
+.latest-heading {
+  color: #f42;
+  font-weight: 700;
+  line-height: 1.1;
+  font-size: 28px;
+  margin-bottom: 18px;
+  margin-top: 24px;
+}
+
+.hard-fork .latest-heading {
+  margin-left: 13px;
+}
+
+.inside-3rd-section {
+  position: relative;
+  left: 11px;
+  top: -170px;
+  width: 70%;
+  font-size: 16px;
+  font-weight: 800;
+}
+
+.inside-3rd-section h2 {
+  margin-top: 6px;
+  margin-bottom: 4px;
+}
+
+.latest-down {
+  color: rgb(248, 58, 25);
+}
+
+.latest-down h4 {
+  text-transform: uppercase;
+}
+
+.latest-down p,
+.col-2 h3 {
+  width: 80%;
+  line-height: 1;
+  font-weight: 700;
+  margin-top: 8px;
+}
+
+.latest-list {
+  margin-top: 20px;
+  width: 95%;
+  list-style: none;
+}
+
+.latest-list li::before {
+  content: "\2022";
+  color: rgb(233, 42, 42);
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+}
+
+.latest-list li {
+  margin-top: 6px;
+  margin-left: 15px;
+}
+
+@media (min-width: 320px) {
```

Comment:
- [ ] No small breakpoint as it is mobile-first and the base styles are for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))

* medium: min-width around 768px
* large: min-width 1024px